framework/workspace/qml/MuseScore/Workspace/NewWorkspaceDialog.qml

```diff
@@ -0,0 +1,165 @@
+import QtQuick 2.9
+import QtQuick.Layouts 1.12
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.Workspace 1.0
+
+QmlDialog {
+    id: root
+
+    width: 552
+    height: 360
+
+    title: qsTrc("workspace", "Create new workspace")
+
+    Rectangle {
+        anchors.fill: parent
+
+        color: ui.theme.popupBackgroundColor
+
+        NewWorkspaceModel {
+            id: workspaceModel
+        }
+
+        Component.onCompleted: {
+            workspaceModel.load()
+        }
+
+        ColumnLayout {
+            anchors.fill: parent
+            anchors.topMargin: 24
+            anchors.leftMargin: 36
+            anchors.rightMargin: 16
+            anchors.bottomMargin: 16
+
+            spacing: 0
+
+            StyledTextLabel {
+                text: qsTrc("workspace", "Create new workspace")
+
+                font.pixelSize: 24
```

Comment:
don't forget to use font from theme after merging #6984

Reply:
fixed